pages/add/add.jsx

```diff
@@ -1,12 +1,201 @@
 import React from 'react';
+import { browserHistory } from 'react-router';
+import HintMessage from '../../components/hint-message/hint-message.jsx';
+import Service from '../../js/service.js';
+import UserData from '../../js/user-data.js';
+import env from "../../config/env.generated.json";
+
+const CreatorInputField = () => {
+  return (
+    <input name="creators" type="text" placeholder="Name" width="150" className="form-control" />
```

Comment:
Right now these are inline blocks, so clicking "add another creator" does not add a new creator as a line, it adds it as if it's text and things look a little funky